mandatory/2-logic-error.js

```diff
@@ -1,16 +1,16 @@
 // The syntax for these functions is valid but there are some errors, find them and fix them
 
 function trimWord(word) {
-  return wordtrim();
+  return word.trim();
 }
 
 function getStringLength(word) {
-  return "word".length();
+  return word.length;
 }
 
 function multiply(a, b, c) {
   a * b * c;
```

Comment:
Question: do we still need this here?

Reply:
@Abdirizak-Mohamed sorry what's it exactly that we don't need here?